### PR TITLE
docs(dropdown): audit style

### PR DESCRIPTION
**Pull request by aagonzales**
Closes https://github.com/carbon-design-system/carbon-website/issues/1997

- Audit fixes to style tab
- Experimented with replacing classnames with friendly names in the tables.

### CODE REVIEW

File: src/pages/components/dropdown/style.mdx [hunk 1]
```diff
@@ -85,103 +143,99 @@ height of a closed dropdown stays consistent while the height of an open
...
 
-![Structure and spacing for a multiselect dropdown](images/dropdown-style-3.png)
+![Dropdown sizes](images/dropdown-style-9.png)
 
 </div>
 
-<Caption>
-  Structure and spacing measurements for a multiselect dropdown | px / rem
-</Caption>
+<Caption>Dropdown sizes | px / rem<</Caption>
```
Comment (alisonjoseph) [inline src/pages/components/dropdown/style.mdx:184]:
```suggestion
<Caption>Dropdown sizes | px / rem</Caption>
```

**Review by laurenmrice — CHANGES_REQUESTED** [review]
Looks awesome!!! I like the friendly name formats in the tables. Just a couple small image tweaks for consistency:

-------


**Multi-select states:**
- Missing alert icon for error state
- First menu option in the selected state dropdown should say "Option 1" to be consistent with the other menus.
multi-select

-------

**Inline states:**
- "Label" in the open focus state should read  "Choose an option"
inline

-------

Sizes:
- The prompt text should have the "..." after the text to be consistent with prompt text in other images.
sizes

-------

**Multi-select spec:**
- The 8px padding bottom indicator in the open state is off center with the padding opacity rectangle.
- The drop shadow does not go all the way to the bottom of the menu
multi-select spec

**Comment by aagonzales** [comment]
omg thank you for always looking at the images so closely @laurenmrice! good catches! They should be fixed now.

**Review by laurenmrice — APPROVED** [review]
Looks good to me ! 🎉

File: src/pages/components/dropdown/style.mdx [hunk 2]
```diff
@@ -12,70 +12,128 @@ Inputs come in two different colors. The default input color is `$field-01` and
...
 
-<div className="image--fixed">
-
-![Normal and opened dropdowns](images/dropdown-style-1.png)
-
-</div>
-
-<Caption>Example of a closed and opened dropdown</Caption>
+<Caption>
+  Dropdown variant examples: default, inline, multi-select, combo-box.
```
Comment (janchild) [inline src/pages/components/dropdown/style.mdx:43]:
```suggestion
  Dropdown variant examples: default, inline, multiselect, combo box.
```

File: src/pages/components/dropdown/style.mdx [hunk 3]
```diff
@@ -12,70 +12,128 @@ Inputs come in two different colors. The default input color is `$field-01` and
...
+
+<Row>
+<Column colLg={12}>
+
+![Dropdown and combo-box states](images/dropdown-style-2.png)
+
+</Column>
+</Row>
+
+<Caption>Dropdown and combo-box states</Caption>
```
Comment (janchild) [inline src/pages/components/dropdown/style.mdx:80]:
```suggestion
<Caption>Dropdown and combo box states</Caption>
```

File: src/pages/components/dropdown/style.mdx [hunk 4]
```diff
@@ -12,70 +12,128 @@ Inputs come in two different colors. The default input color is `$field-01` and
...
+
+<Row>
+<Column colLg={12}>
+
+![Multi-select dropdown states](images/dropdown-style-3.png)
+
+</Column>
+</Row>
+
+<Caption>Multi-select dropdown states</Caption>
```
Comment (janchild) [inline src/pages/components/dropdown/style.mdx:90]:
```suggestion
<Caption>Multiselect dropdown states</Caption>
```

File: src/pages/components/dropdown/style.mdx [hunk 5]
```diff
@@ -12,70 +12,128 @@ Inputs come in two different colors. The default input color is `$field-01` and
...
+
+<Row>
+<Column colLg={12}>
+
+![Inline dropdown states](images/dropdown-style-4.png)
+
+</Column>
+</Row>
+
+<Caption>Inline dropdown states.</Caption>
```
Comment (janchild) [inline src/pages/components/dropdown/style.mdx:122]:
```suggestion
<Caption>Inline dropdown states</Caption>
```

File: src/pages/components/dropdown/style.mdx [hunk 6]
```diff
@@ -85,103 +143,99 @@ height of a closed dropdown stays consistent while the height of an open
...
+| Label      | margin-bottom | 8 / 0.5  | `$spacing-03` |
+| Field      | padding-left  | 16 / 1   | `$spacing-05` |
+|            | padding-right | 72 / 4.5 | –             |
+| Chevron    | padding-right | 16 / 1   | `$spacing-05` |
+| Clear icon | padding-right | 8 / .5   | `$spacing-03` |
 
 <div className="image--fixed">
 
-![Structure and spacing for inline dropdown](images/dropdown-style-5.png)
+![Spacing for combo-box dropdown](images/dropdown-style-6.png)
```
Comment (janchild) [inline src/pages/components/dropdown/style.mdx:198]:
```suggestion
![Spacing for combo box dropdown](images/dropdown-style-6.png)
```

File: src/pages/components/dropdown/style.mdx [hunk 7]
```diff
@@ -85,103 +143,99 @@ height of a closed dropdown stays consistent while the height of an open
...
 
 </div>
 
-<Caption>Structure and spacing for inline dropdown | px / rem</Caption>
+<Caption>Spacing for combbo-box dropdown | px / rem</Caption>
```
Comment (janchild) [inline src/pages/components/dropdown/style.mdx:202]:
```suggestion
<Caption>Spacing for combo box dropdown | px / rem</Caption>
```

**Review by janchild — COMMENTED** [review]
@aagonzales just a few little content tweaks here. :) 
Multi words don't have a hyphen; they're joined into one word. And combo box is two words.

**Review by janchild — APPROVED** [review]
🎉

**Review by dakahn — APPROVED** [review]

**Review by aledavila — APPROVED** [review]

**Pull request merged**